packages/cursorless-vscode/src/keyboard/KeyboardCommandsModal.ts

```diff
@@ -108,6 +108,8 @@ export default class KeyboardCommandsModal {
 
   modeOn = async () => {
     if (this.isModeOn()) {
+      // Set target to current selection if we re-run mode on
```

Comment:
don't follow this comment (can't even quite parse it)

Reply:
While trying to write a better comment, I decided I didn't like the way it was implemented. See https://github.com/cursorless-dev/cursorless/pull/2169/files#diff-9c79f806edb4ea294501b9810d847bd61a9cb17bcfe6930f91ca4cf062a36fc5 for my new solution